src/equations/compressible_euler_multicomponent_2d.jl

```diff
@@ -175,14 +186,14 @@ function initial_condition_shock_bubble(x, t, equations::CompressibleEulerMultic
   # bubble test case, see Agertz et al. https://arxiv.org/pdf/1904.00972
   # other reference: https://www.researchgate.net/profile/Pep_Mulet/publication/222675930_A_flux-split_algorithm_applied_to_conservative_models_for_multicomponent_compressible_flows/links/568da54508aeaa1481ae7af0.pdf
   # typical domain is rectangular, we change it to a square, as Trixi can only do squares
-  @unpack gas_constant1, gas_constant2 = equations
+  @unpack gas_constant = equations
 
   # Positivity Preserving Parameter, can be set to zero if scheme is positivity preserving
   delta   = 0.03
 
-  # Region I 
+  # Region I
   rho1_1  = delta
-  rho2_1  = 1.225 * gas_constant1/gas_constant2 - delta
+  rho2_1  = 1.225 * gas_constant[1]/gas_constant[2] - delta
```

Comment:
It looks like you assume that you use two components in this function. Hence, you should restrict the type of the equations to the two-component model (using something like `equations::CompressibleEulerMulticomponentEquations2D{2}`).

Reply:
done in 4c720b0